CHANGELOG.md

```diff
@@ -29,6 +29,23 @@
 
 ### Jsonnet
 
+* [CHANGE] [Common storage configuration](https://grafana.com/docs/mimir/v2.3.x/operators-guide/configure/configure-object-storage-backend/#common-configuration) is now used to configure object storage in all components. This is a breaking change in terms of Jsonnet manifests and also a CLI flag update for components that use object storage, so it will require a rollout of those components. The changes include: #3257
```

Comment:
```suggestion
* [BREAKING CHANGE] [Common storage configuration](https://grafana.com/docs/mimir/v2.3.x/operators-guide/configure/configure-object-storage-backend/#common-configuration) is now used to configure object storage in all components. This is a breaking change in terms of Jsonnet manifests and also a CLI flag update for components that use object storage, so it will require a rollout of those components. The changes include: #3257
```

Reply:
I'm not sure that it's an option in our changelog. I think that all changes are assumed at least _breakish_, otherwise they're not considered changes.